Added some explanation to Excel Row 1 - 3 for better understanding

nh916 opened the pull request.

nh916 committed:
changed wording for documentation

individual sheets.md: changed "Value Type" to Row 4 - ∞ value type because that makes more sense to me

individual sheets.md: changed "refer to key table" to literally saying "refer to link"

requirements_docs.txt: updated mkdocs mkdocs-material

nh916 committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	docs/individual sheets.md

nh916 committed:
added some clarification to Excel Row 1 - 3 in plain english

added needed css to extra.css

rjmello commented on docs/excel rows.md:
```diff
@@ -23,13 +23,64 @@ sheet are treated differently than Rows 4 - ∞.
 - <span class="row-1">identifier</span>
-  - Column with key-value pairs for material identifiers
+  - Column that can identify a row
+    - Example in everyday life can be `social security` that can identify a single citizen
```
I don't think this sentence is necessary.

nh916 replied:
I will remove lines 26 and 27 as chemists already know what material identifiers are

rjmello commented on docs/excel rows.md:
```diff
@@ -23,13 +23,64 @@ sheet are treated differently than Rows 4 - ∞.
   - The condition under which the property was discerned
 - <span class="row-1">identifier</span>
-  - Column with key-value pairs for material identifiers
+  - Column that can identify a row
```
I suggest leaving the original sentence but keeping your examples.

nh916 committed:
made some slight clarifications

excel rows.md: added an example for identifier row

mkdocs.yml: changed Navigation panel from "Excel Sheets Rows" to something that makes more sense "Structure of Excel Sheets"

individual sheets.md: under materials section I also cleared up that materials can be either for ingredients or process product
